src/HotChocolate/Marten/src/Data/Filtering/Extensions/RequestExecutorBuilderExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using HotChocolate.Data.Filters;
+using HotChocolate.Execution.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace HotChocolate.Data.Marten.Filtering.Extensions;
+
+public static class RequestExecutorBuilderExtensions
+{
+    public static IRequestExecutorBuilder AddMartenFiltering(this IRequestExecutorBuilder requestExecutorBuilder)
```

Comment:
@jabellard  Does marten have a native api too? if so we might want to change this to `AddMartenQueryableFiltering`. Because then we would still be able to add the native filtering stuff too.
Also wonder what @michaelstaib  thinks about it. specfic or less specific

Reply:
@PascalSenn no. There's no other api.